src/users/04-moderation.md

```diff
@@ -31,3 +31,229 @@ All moderation actions are taken on the context menu of posts or comments. Click
+
+```
+psql -U lemmy
+```
+
+For example
+
+```
+postgres:/# psql -U lemmy
+psql (15.5)
+Type "help" for help.
+
+lemmy=# 
+```
+
+Query for the image by the \"alias\" (the filename)
+
+```
+select * from image_upload where pictrs_alias = '<image_filename>';
+```
+
+For example
+
+```
+lemmy=# select * from image_upload where pictrs_alias = '001665df-3b25-415f-8a59-3d836bb68dd1.webp';
+ local_user_id | pictrs_alias | pictrs_delete_token | published 
+---------------+--------------+---------------------+-----------
+1149 | 001665df-3b25-415f-8a59-3d836bb68dd1.webp | d88b7f32-a56f-4679-bd93-4f334764d381 | 2024-02-07 11:10:17.158741+00
+(1 row)
+
+lemmy=# 
+```
+
+Now, take the \``pictrs_delete_token`\` from the above output, and use
+it to delete the image.
+
+The following command should be able to be run on any computer connected
+to the internet.
+
+```
+curl -i "https://<instance_domain>/pictrs/image/delete/<pictrs_delete_token>/<image_filename>"
+```
+
+For example:
+
+```
+user@disp9140:~$ curl -i "https://monero.town/pictrs/image/delete/d88b7f32-a56f-4679-bd93-4f334764d381/001665df-3b25-415f-8a59-3d836bb68dd1.webp"
+
+HTTP/2 204 No Content
+server: nginx
+date: Fri, 09 Feb 2024 15:37:48 GMT
+vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
+cache-control: private
+referrer-policy: same-origin
+x-content-type-options: nosniff
+x-frame-options: DENY
+x-xss-protection: 1; mode=block
+X-Firefox-Spdy: h2
+user@disp9140:~$ 
+```
+
+> ⓘ Note: If you get an \``incorrect_login`\` error, then try \[a\]
+logging into the instance in your web browser and then \[b\] pasting the
+\"`https://<instance_domain>/pictrs/image/delete/<pictrs_delete_token>/<image_filename>`\"
+URL into your web browser.
+
+The image should be deleted.
+
+##### Method Two: /internal/purge?alias={alias}
+
+Alternatively, you could execute the deletion directly inside the pictrs
+container. This eliminates the need to fetch the \``delete_token`\`.
+
+First, open a shell on your running \``pictrs`\` container. If you
+installed Lemmy with docker compose, use \``docker compose ps`\` to get
+the \"SERVICE\" name of your postgres host, and then enter it with
+\``docker exec`\`
+
+```
+docker compose ps --format "table {{.Service}}\t{{.Image}}\t{{.Name}}"
+docker compose exec <docker_service_name> /bin/sh
+```
+
+For example:
+
+```
+user@host:/home/user/lemmy# docker compose ps --format "table {{.Service}}\t{{.Image}}\t{{.Name}}"
+SERVICE    IMAGE                            NAME
+lemmy      dessalines/lemmy:0.19.3          lemmy-lemmy-1
+lemmy-ui   dessalines/lemmy-ui:0.19.3       lemmy-lemmy-ui-1
+pictrs     docker.io/asonix/pictrs:0.5.4    lemmy-pictrs-1
+postfix    docker.io/mwader/postfix-relay   lemmy-postfix-1
+postgres   docker.io/postgres:15-alpine     lemmy-postgres-1
+proxy      docker.io/library/nginx          lemmy-proxy-1
+user@host:/home/user/lemmy# 
+
+user@host:/home/user/lemmy# docker compose exec pictrs /bin/sh
```

Comment:
The name is always `pictrs`, no need for ps command.

Reply:
I think redundancy is good in documentation for clarity.